### PR TITLE
Junkheap Mommi

### PR DESCRIPTION
**Pull request by BurntDevil**
[<sprites>]
Not all MoMMIs get to be high tech crabs. Sometimes, you just gotta work with what you have on hand. 

Below is footage of an example of such a model. Please do not open the crab.
https://webm.red/LIQP.webm

:cl:
 * rscadd: NT ran out of some MoMMI parts, a batch of special MoMMIs were shipped out with what was available. Please do not bully the toolboxes.

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
Hello can my bot come inside the SM room? It has autism

File: code/modules/mob/living/silicon/mommi/mommi_modules.dm [hunk 1]
```diff
@@ -85,4 +86,4 @@
 	sprites = list(
 		"Gravekeeper" = "cogspider"
 		)
-	law_type = "Gravekeeper"
```
Comment (boy2mantwicethefam) [inline code/modules/mob/living/silicon/mommi/mommi_modules.dm:88]: I think an indentation was removed here

Comment (BurntDevil) [reply code/modules/mob/living/silicon/mommi/mommi_modules.dm:88]: Does look like that happened somehow. Fixing.

**Comment by FudgePucker** [comment]
How do hats look on it? What does the emag version look like?

**Comment by BurntDevil** [comment]
Hats are kinda a mixed bags due to the camera but thats just the nature of the sprite. The emagged version is just a red screen instead of the blue - wasnt in the mood to make the box the syndie box. Probably do that down the road after some other projects are done.

File: code/modules/mob/mommi_modules.dm [hunk 2]
```diff
@@ -0,0 +1,89 @@
+
+/obj/item/weapon/robot_module/mommi
```
Comment (PaulRitter) [inline code/modules/mob/mommi_modules.dm]: am i having a stroke or is this an exact copypaste from code/modules/mob/living/silicon/mommi/mommi_modules.dm?

**Review by PaulRitter — CHANGES_REQUESTED** [review]

File: code/modules/mob/mommi_modules.dm (hunk 2 above)
Comment (jknpj) [reply code/modules/mob/mommi_modules.dm]: Yes, it is

Comment (PaulRitter) [reply code/modules/mob/mommi_modules.dm]: pfew got worried there for a sec

also good on @boy2mantwicethefam for not noticing it in his review
ya doofus

**Comment by brndd** [comment]
shouldn't the camera rotate along with the MoMMI?

File: code/modules/mob/mommi_modules.dm (hunk 2 above)
Comment (BurntDevil) [reply code/modules/mob/mommi_modules.dm]: >is this line from code/modules/mob/mommi_modules.dm  the same as code/modules/mob/living/silicon/mommi/mommi_modules.dm?
Are you sure you arent having a stroke? What is issue?

Comment (brndd) [reply code/modules/mob/mommi_modules.dm]: The issue is that why do we need this new file? What is its purpose? It's not even ticked in vgstation.dme from what I can tell.

Comment (BurntDevil) [reply code/modules/mob/mommi_modules.dm]: I dont understand, I just added my line to the existing mommi_modules.dm, I didnt create a new file?

Comment (BurntDevil) [reply code/modules/mob/mommi_modules.dm]: i ams retard

Comment (PaulRitter) [reply code/modules/mob/mommi_modules.dm]: like how does that even happen

File: code/modules/mob/living/silicon/mommi/mommi_modules.dm (hunk 1 above)
Comment (PaulRitter) [reply code/modules/mob/living/silicon/mommi/mommi_modules.dm:88]: still broken btw, ya didn't fix it

**Comment by Coxswain-Navigator** [comment]
Can it be a grey toolbox instead of a yellow one

**Pull request merged**